frontend/src/components/Bootstrap/index.tsx

```diff
@@ -19,26 +21,33 @@ type Form = {
 
 type State = {
   isInProgress: boolean;
+  isSubmitInProgress: boolean;
 };
 
 export default function Bootstrap() {
+  const [isAlreadyBootstrapped, setIsAlreadyBootstrapped] = useState<boolean>(false);
+  const [isBootstrapped, setIsBootstrapped] = useState<boolean>(true);
```

Comment:
Naming is a little bit confusing.`isBootstrapped` might be `isComplete`, also what do you say about merging everything in a single state?